src/Clients/Event/Entities/HttpStateEntity.php

```diff
@@ -22,16 +22,17 @@ final class HttpStateEntity
 
     public function __construct(
         public readonly RequestInterface $request,
-    )
-    {
+    ) {
         $this->start = Time::seconds();
-        $this->id = md5($this->start . $this->request->getUri());
+        $this->id = substr(md5($this->start . $this->request->getUri()), 0, 16);
```

Comment:
říkal jsem si že id pro request může být kratší, stále to 1.8*10^19 možností a zkrátí se tím název souboru